pkg/rancher-desktop/backend/k3sHelper.ts

```diff
@@ -321,11 +323,20 @@ export default class K3sHelper extends events.EventEmitter {
     }
 
     // Check that this release has all the assets we expect.
-    if (Object.values(this.filenames).every(name => entry.assets.some(v => v.name === name))) {
-      console.debug(`Adding version ${ version.raw }`);
-      this.versions[version.version] = new VersionEntry(version);
-    } else {
-      console.debug(`Skipping version ${ version.raw } due to missing files`);
+    if (entry.assets.find(ea => ea.name === this.filenames.exe)) {
+      if (entry.assets.find(ea => ea.name === this.filenames.checksum)) {
+        let haveImage = false;
+
+        for (let index = 0; !haveImage && index < this.filenames.images.length; index++) {
+          if (entry.assets.find(ea => ea.name === this.filenames.images[index])) {
+            haveImage = true;
+            this.versions[version.version] = new VersionEntry(version);
+            console.log(`Adding version ${ version.raw } - ${ this.filenames.images[index] }`);
+          } else {
+            console.debug(`Skipping version ${ version.raw } due to missing files`);
+          }
+        }
+      }
     }
```

Comment:
> downside is that we get a worse log message (no idea which image we will use).

Can you explain that? Maybe give an example?

Reply:
> > downside is that we get a worse log message (no idea which image we will use).
> 
> Can you explain that? Maybe give an example?

The old message:
> `Adding version ${ version.raw } - ${ this.filenames.images[index] }`

With my (first) suggestion):
> `Adding version ${ version.raw }`

My suggested version drops the second part.

Reply:
I like the 'foundImage' suggestion, and have made that change.  I'd like to keep the log messages for now since they are related to the other changes in this P/R.